ui/src/containers/HeaderContainer.js

```diff
@@ -19,30 +24,36 @@ const HeaderContainer = ({ currentUser, history }) => {
   const logout = () => {
     window.location.replace('/oauth/sign_in');
   };
-
-  return (
-    <Header
-      currentUser={currentUser}
-      appName={config.app.name.toUpperCase()}
-      isDropDownOpen={isDropDownOpen}
-      onUserDropdownToggle={onUserDropdownToggle}
-      onTitleClick={onTitleClick}
-      logout={logout}
-    />
-  );
+  if (isUserRequestFailed) {
```

Comment:
I think this might solve the issue we saw on MDC where the screen loads with no data if the URL is manipulated after logout. wdyt @davidffrench @wei-lee